assets/css/style.css

```diff
@@ -70,10 +70,18 @@ input#height {
 	width: 3em;
 }
 
+[mv-multiple="resolution"] {
+	margin: 0 .2em 0 .2em;
```

Comment:
Super minor but you don't need to repeat values here, `0 .2em` would get expanded to `0 .2em 0 .2em` anyway.

From https://developer.mozilla.org/en-US/docs/Web/CSS/margin : 

> - When one value is specified, it applies the same margin to all four sides.
> - When two values are specified, the first margin applies to the top and bottom, the second to the left and right.
> - When three values are specified, the first margin applies to the top, the second to the left and right, the third to the bottom.
> - When four values are specified, the margins apply to the top, right, bottom, and left in that order (clockwise).